src/localStorage.js

```diff
@@ -0,0 +1,75 @@
+import SketchfabIntegration from "./sketchfabIntegration.js";
+import { modelLayer } from './layers.js';
+import {
+    addModel,
+} from '../index.js';
+const sketchfabIntegration = new SketchfabIntegration();
```

Comment:
This may cause problems now or in the future, creating another instance of `sketchfabIntegration` to use here. 

This instance holds a copy of the token `sketchfabIntegration.token`. When you create a fresh new instance, the token is not defined (until you call `sketchfabIntegration.checkToken()`).

This may not be causing any issues now because the SketchfabIntegration class also stores the token in local storage and this file gets it from local storage.

It would probably be better to have `localStorageSetUp` take a reference to the `sketchfabIntegration` instance that is created in `index.js` and that the rest of the application uses.

It would be _even better_ if this `localStorage` file didn't require using sketchfabIntegration. Because it can be confusing if multiple pieces of the app are making these calls that affect the whole app. If this were a larger app it would be better to have `index.js` handle calling the necessary sketchfabIntegration calls before calling the `localStorage` calls. 

Similarly for the `getSketchfabModelFromLocalStorage` function, it doesn't need to actually fetch the model and add it directly to the scene. That function could just return the loaded info from local storage, and it's up to `index.js` to figure out how to download this model, and how to add it to the scene.

This decoupling is very useful for larger applications because it means it would be very easy to (1) re-use this class in other projects that need some local storage saving (2) easily swap out whether we load the models from Sketchfab or another source (3) easily add the models to ThreeJS/Cesium/another engine, since that code for adding it would be centralized in index.js and not in multiple places.